### ISSUE TITLE
Windows 10: NVDA missing from the Ease of Access Center

### ISSUE THREAD

**Issue by elliott94** [nvaccess/nvda#7775]
Several releases ago, I'm aware that some changes were made so that NVDA was included into the Ease of Access Center. When opening EoA and navigating around the available options, however, the entry for NVDA appears to be missing. Is this a known issue?

**Comment by leonardder** [comment]
NVDA is available here in the sign in settings from the ease of access center.

**Comment by elliott94** [comment]
Thanks for confirming this. Did you just hit Windows + U to get to this screen?

**Comment by Adriani90** [comment]
This is not appearing there anymore, you will just find the narator settings there. I guess Microsoft now wants to promote his own screen reader so it makes sense to exclude NVDA and other screen readers there. Note that you can still find NVDA on your desktop, quick launch area, start menu and what not. If you still like NVDA in that ease of access center, this is something which should be discussed directly with Microsoft.
I am closing this issue for now.

**Comment by josephsl** [comment]
Hi, actually, it does – you might be looking at Settings/Ease of Access, which does not include certain advanced settings from Control Panel (if you do open Control Panel/Ease of Access/ Use the computer without a display Screen, you’ll see NVDA listed). Closing as “works for me”. Thanks.
